package.json

```diff
@@ -6,13 +6,13 @@
   "scripts": {
     "clean": "rimraf dist",
     "clean-example": "rimraf src/example/dist",
-    "build": "npm run clean && NODE_ENV=build node build.js",
+    "build": "npm run clean && cross-env NODE_ENV=build node build.js",
     "build-example": "npm run clean-example && npm run build:webpack",
-    "build:webpack": "NODE_ENV=production webpack --config webpack.config.prod.js",
+    "build:webpack": "cross-env NODE_ENV=production webpack --config webpack.config.prod.js",
     "start": "node devServer.js",
     "lint": "eslint src",
-    "test": "./node_modules/karma/bin/karma start --single-run --no-auto-watch",
-    "test:watch" : "./node_modules/karma/bin/karma start"
+    "test": "karma start --single-run --no-auto-watch",
+    "test:watch": "karma start"
```

Comment:
It's not using global.  `npm run` behavior is to automatically add `./node_modules/.bin` to the path, so this is running the locally-installed `./node_modules/.bin/karma` which symlinks `./node_modules/karma/bin/karma` but is cleaner.